src/extension/background-script/actions/lnurl/auth.ts

```diff
@@ -94,6 +95,12 @@ export async function authFunction({
         authResponse?.data?.reason || "Auth: Something went wrong"
       );
     } else {
+      PubSub.publish(`albyEvent.auth`, {
```

Comment:
I think those should subscribe to the existing events. 
having a pubsub publish for each thing that should happen does not really make sense I think

The idea of a pubsub is that I publish an event once (e.g. `lnurl.auth.success` in line 104) and then different subscribes subscribe to that. e.g. one that shows a notification and one that persists something in the DB.